File: 17/TeamBuilder.dart
import 'dart:io';

void main(){
  int friends = int.parse(stdin.readLineSync()!);
  int teams = int.parse(stdin.readLineSync()!);

  int leftOut = friends%teams;
  int teamMembers = ((friends-leftOut)/teams).toInt();

  print("The number of friends in each team is $teamMembers and left out is $leftOut");
}

File: 17/TeamBuilder.py

def main():
    friends = int(input())
    teams = int(input())

    leftOut = friends%teams
    teamMembers = (friends-leftOut)//teams

    print(f"The number of friends in each team is {teamMembers} and left out is {leftOut}")


if __name__ == "__main__":
    main()

